content/docs/next-release/troubleshooting.md

```diff
@@ -81,4 +81,8 @@ The first two metrics should have similar values for the same service. Similarly
 
 ## Istio: missing spans
 
-When deploying your application as part of a service mesh like Istio, the number of moving parts increases significantly and might affect how (and which) spans are reported. If you expect to see spans generated by Istio but they aren't being visible in the Jaeger UI, check the troubleshooting guide on [Istio's website](https://istio.io/faq/distributed-tracing/#no-tracing).
\ No newline at end of file
+When deploying your application as part of a service mesh like Istio, the number of moving parts increases significantly and might affect how (and which) spans are reported. If you expect to see spans generated by Istio but they aren't being visible in the Jaeger UI, check the troubleshooting guide on [Istio's website](https://istio.io/faq/distributed-tracing/#no-tracing).
+
+## Run debug images of the backend components
+
+We provide debug images for each Jaeger component. These images have [delve](https://github.com/go-delve/delve) and respective jaeger component compiled with optimizations disabled. When you run these images, delve triggers the execution of the jaeger component as its child process and immediately attaches to it to begin a new debug session and start listening on TCP port 12345 for remote connections. You can then use your IDEs like [Visual Studio Code](https://code.visualstudio.com/) or [GoLand](https://www.jetbrains.com/go/) to connect to this port and attach with it remotely and perform [debugging](https://golangforall.com/en/post/go-docker-delve-remote-debug.html) by adding breakpoints.
```

Comment:
Looks good. Capitalize names (Jaeger instead of jaeger, for instance) and it looks good. Perhaps add an example for VS Code? Here's what worked for me:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch remote",
            "type": "go",
            "request": "attach",
            "mode": "remote",
            "remotePath": "",
            "port": 12345,
            "host": "127.0.0.1",
            "cwd": "${workspaceRoot}",
        }
    ]
}
```